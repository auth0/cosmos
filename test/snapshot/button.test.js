import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from '@auth0/cosmos'
import 'jest-styled-components'

describe('Button', () => {
  it('renders with automation attribute', () => {
    const tree = renderer.create(<Button />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
