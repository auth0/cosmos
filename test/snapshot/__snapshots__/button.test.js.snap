// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders with automation attribute 1`] = `
.c0 {
  display: inline-block;
  min-height: 42px;
  line-height: 42px;
  min-width: 96px;
  box-sizing: border-box;
  text-transform: uppercase;
  text-align: center;
  white-space: nowrap;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 13px;
  font-weight: 500;
  background: #F1F1F1;
  border: 1px solid;
  border-color: #F1F1F1;
  border-radius: 3px;
  color: #333;
  padding: 0 16px;
  opacity: 1;
  cursor: pointer;
  -webkit-transition: border-color 0.25s,background 0.25s;
  transition: border-color 0.25s,background 0.25s;
}

.c0 .sc-bwzfXH,
.c0 .sc-dnqmqq {
  position: relative;
  top: -1px;
  margin-right: 0;
}

.c0 .sc-bwzfXH {
  color: #333;
}

.c0:hover {
  color: #333;
  background: #E9E8E8;
  border-color: #E9E8E8;
}

.c0:focus {
  background: #E9E8E8;
  border-color: #E9E8E8;
  outline: none;
}

.c0:active {
  background: #DADADA;
  border-color: #DADADA;
  outline: none;
}

<button
  className="c0"
  data-cosmos-key="button"
  disabled={false}
  icon={null}
  size="default"
/>
`;
